src/plugins/timeConductor/ConductorInputsFixed.vue

```diff
@@ -131,6 +131,10 @@ export default {
             this.setViewFromBounds(bounds);
         },
         clearAllValidation() {
+            if (!this.$refs.startDate || !this.$refs.endDate) {
```

Comment:
I'm not a 100% on what this does, but is there any case where we'd want to `clearValidationForInput` for just one of them? So, say there's a startDate, but no endDate, would we want to clear validation for just startDate? If so, maybe build the array below based on if there's is a date.

Reply:
OK I can do that.

Reply:
Done